torch/csrc/inductor/aoti_torch/shim_cuda.cpp

```diff
@@ -0,0 +1,28 @@
+
+#include <torch/csrc/inductor/aoti_torch/c/shim.h>
+
+#include <c10/cuda/CUDAStream.h>
+
+#include <iostream>
+
+#define CONVERT_EXCEPTION_TO_ERROR_CODE(...)                 \
+  try {                                                      \
+    __VA_ARGS__                                              \
+  } catch (const std::exception& e) {                        \
+    std::cerr << "Error: " << e.what() << std::endl;         \
```

Comment:
Wondering if this should be sent to some real logging framework 🤔  but what about ABI .......

Reply:
oh but this is in the predictor, you have access to everything as needed